assets/section-main-product.css

```diff
@@ -1322,6 +1323,16 @@ a.product__text {
   margin-bottom: var(--icon-size);
 }
 
+/* SKU block */
+
+.product__sku.visibility-hidden {
+  display: block;
+}
+
```

Comment:
```suggestion
```

Do we need this? I think `p` elements have `display: block` by default, but maybe there's a use case I haven't considered.

Reply:
Actually you are right. I think we are good to remove it.